lib/RawData.cpp

```diff
@@ -30,6 +31,77 @@ namespace motioncam {
         128
     };
 
```

Comment:
Could you please use this header only library for the SIMD code?
https://github.com/simd-everywhere/simde

This would make my life a bit easier as I am on an ARM mac.